README.md

```diff
@@ -73,12 +74,23 @@ Semantic shapes are frequently described using the SHACL or ShEx language.
 
 ## Shape Generators
 
+ - [SHACL Play!](https://shacl-play.sparna.fr/play/convert) - This utility applies a set of SHACL rules to convert an RDF dataset into another structure. Some SHACL rules are provided in the catalog to convert OWL ontologies to SHACL. 
 - [owl2shacl](https://github.com/sparna-git/owl2shacl) - OWL-to-SHACL conversion rules.
+- [TopQuadrant: OWL Axioms to SHACL Constraints](https://www.topquadrant.com/doc/latest/reference/Transform_ConvertOWLAxiomsToSHACLConstraints.html) - Commercial closed-source solution by TopQuadrant
+- [elevont/owl2shacl](https://github.com/elevont/owl2shacl) - 
+[![License: AGPL-3.0-or-later](
+    https://img.shields.io/badge/License-AGPL--3.0--or--later-blue.svg)](
+    LICENSE.txt)
+[![Package Releases](
+    https://img.shields.io/crates/v/owl2shacl.svg)](
+    https://crates.io/crates/owl2shacl)
+A CLI tool that tries to convert simple OWL ontologies into SHACL shapes.
 
 ## Shape-based Query Generators
 
 - [shape-to-query](https://github.com/hypermedia-app/shape-to-query) <img alt="NPM Version" src="https://img.shields.io/npm/v/@hydrofoil/shape-to-query" align="top"> <img alt="GitHub Release Date" src="https://img.shields.io/github/release-date/hypermedia-app/shape-to-query" align="top"> - Generate SPARQL queries from SHACL Shapes; [docs](https://shape-to-query.hypermedia.app/docs); `MIT` license; `Typescript`.
-  - [playground](https://shape-to-query.hypermedia.app)
+  - [playground](https://shape-to-query.hypermedia.app) - A CLI tool that tries to convert simple OWL ontologies into [SHACL] shapes.
+- [SHACL Play!](https://shacl-play.sparna.fr/play/sparql) - This is a SHACL-based SPARQL generator. It generates SPARQL queries used to extract a subset of data from a knowledge graph, based on the SHACL specification of the target dataset structure.
```

Comment:
Change to 
SHACL Play! SPARQL: generates SPARQL queries to extract RDF subset based on SHACL specification of the target dataset.